<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="l_fly">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>fly.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_trap">
    <property name="geometry">
     <rect>
      <x>929</x>
      <y>10</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>trap.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_cactus">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cactus.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_cactus_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cactus.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_cactus_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>140</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cactus.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_cactus_4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>340</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cactus.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_cactus_5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cactus.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_cactus_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>170</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cactus.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_cactus_7">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>350</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cactus.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_fly2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>fly2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_fly3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>fly3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_window">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>450</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>window.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
